index.html

```diff
@@ -9,6 +9,50 @@
 </head>
 <body>
     <main>
+        <section id="popup-window">
+            
+            <div class="modal" id="modal">
+             <div id="projectDetailsPopup" class="popup">
+                 <span class="close-button" id="close-popup">&times;</span>
+                 <h4 id="popupName"></h4>
+                 <ul class="list">
+                     <li class="canopy" id="client"></li>
+                     <li class="bullet" id="role"></li>                        
+                     <li class="bullet" id="year"></li>
+                 </ul>
+                 <img id="popupImage" src="" alt="projectImage">
+                 <div class="flexPop">
+                    <div class="con1">
+                        <p id="popupDescription"></p>
+                     </div>
+                     <div class="con2">
+                        <ul class="buttonList" id="techList">
+                            <li><button class="smallbtn" id="tech1"></button> </li>
+                            <li><button class="smallbtn" id="tech2"></button> </li>
+                            <li><button class="smallbtn" id="tech3"></button> </li>
+                        </ul>
+                        <hr id="bottomLine">
+                        <div class="liveButton">
+                           <a  href="" class="buttonlrg"  id="popupLive">
+                           See Live
+                           <img src="img/live.png" alt="See Live Icon">
+                         </a>
+                         <a  href="" class="buttonlrg" id="popupSource">
+                           See Source
+                           <img src="img/Icon.png" alt="See Source Icon">
+                         </a>
```

Comment:
- [ ] Since the "See Live" and "See source" link depends on each project I'd suggest you load it dynamically from the projects array instead of hard coding them here because whenever you add a new project to the array in the future the link should redirect to that project and the above implementation will prevent that. ✔

Reply:
@AbelG101 Thank you for the valuable review. I have already assigned the links dynamically.https://github.com/Porkalaikarthika/Portfolio/pull/11/files#diff-5fdf819a01be165666bd809fc87f7fe26de6edf815aeff34d05ae163c9960a1bR82-R83
 Do I have to still make changes